src/screens/TopicPageScreen/index.js

```diff
@@ -68,11 +69,16 @@ const TopicPageScreen = (props) => {
     ? feedsContext.topicFeeds.filter((feed) => feed?.topics?.includes(topicName))
     : [];
   const mainFeeds = feedsContext.feeds;
+  const {timer, viewPostTimeIndex} = feedsContext;
+
   const [offset, setOffset] = React.useState(0);
   const [client] = React.useContext(Context).client;
   const refBlockComponent = React.useRef();
   const scrollY = React.useRef(new Animated.Value(0)).current;
   const {mappingColorFeed} = useCoreFeed();
+
+  const {onWillSendViewPostTime} = useViewPostTimeHook(dispatch, timer, viewPostTimeIndex);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The extraction of `timer` and `viewPostTimeIndex` from `feedsContext` and the usage of `useViewPostTimeHook` are correct. These changes are necessary to implement the view post time feature in the topic feed.
```diff
+ const {timer, viewPostTimeIndex} = feedsContext;
+ const {onWillSendViewPostTime} = useViewPostTimeHook(dispatch, timer, viewPostTimeIndex);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed